pom.xml

```diff
@@ -24,6 +24,7 @@
 
   <modules>
     <module>sender-sqs</module>
+    <module>sender-sqs2</module>
```

Comment:
how about `sender-awssdk-sqs`? I was recently looking around for a way to make sense of the different versions and not get into the marketing vs artifact wars. If you plop this search it surprisingly works https://search.maven.org/search?q=awssdk-sqs

Reply:
I like that better.  thanks for the suggestion!